samcli/commands/build/click_container.py

```diff
@@ -0,0 +1,27 @@
+"""
+Module to check container based cli parameters
+"""
+import click
+
+
+class Container(click.Option):
```

Comment:
Should we name this something more specific like `ContainerOptions`? We have other Container classes and might be easier to read/understand the difference at a glance.

Reply:
Sure I will change the name